### PR TITLE
Migrated to an IsDesktop Context

### PR DESCRIPTION
**Pull request by G3sit**

### CODE REVIEW

**Comment by G3sit** [comment]
[isDesktop zu einem Context umschreiben](https://trello.com/c/nlumgvhk/516-isdesktop-zu-einem-context-umschreiben)

File: src/components/platform/nav/Nav.tsx [hunk 1]
```diff
@@ -275,32 +275,37 @@ class Nav extends Component<RouteComponentProps, NavState> {
...
-                                        <Dropdown.Item as={NavLink} onClick={this.navOnClick} to={"/about-us"}
-                                                       role={"button"}>
-                                            <FAE icon={faInfoCircle}/>&nbsp;
-                                            Über uns
-                                        </Dropdown.Item>
-                                    </NavDropdown>
-                                </BootstrapNav>
-                            )}
+                            <DesktopContext.Consumer children={isDesktop => {
+                                if(isDesktop) {
```
Comment (ma1160) [inline src/components/platform/nav/Nav.tsx]: this condition must be reversed

**Review by nic-schi — APPROVED** [review]
Approved. The wrong boolean was fixed in the new commit.

**Review by ma1160 — APPROVED** [review]

File: src/components/platform/nav/Nav.tsx (hunk 1 above)
Comment (nic-schi) [reply src/components/platform/nav/Nav.tsx]: was changed in new commit 👍

**Pull request merged**